docs/INSTALL.md

```diff
@@ -135,3 +135,18 @@ export GDOCK_PATH=some-folder
 instructions as here only the model generation is used. Please check the
 [repository page](https://github.com/rvhonorato/gdock) for more
 information.
+
+## `openmm`
+
+1. Install the latest version of openmm:
```

Comment:
Since we have all in the same `## openmm` header, what do you think about suggesting a single command? `conda install -c conda-forge openmm pdbfixer`? It would be easier for `conda` to evaluate requirements also, i think.

Reply:
yes, agreed